code/modules/clothing/outfits/ert.dm

```diff
@@ -150,9 +150,9 @@
 	id_trim = /datum/id_trim/centcom/official
 	uniform = /obj/item/clothing/under/rank/centcom/official
 	back = /obj/item/storage/backpack/satchel
+	box = /obj/item/storage/box/survival
```

Comment:
This isn't related to the CTF changes, you should put these fixes in their own separate PR so they aren't held up by the balance aspects of this PR.